geode-core/src/main/java/org/apache/geode/internal/cache/tx/DistributedTXRegionStub.java

```diff
@@ -146,9 +167,8 @@ public Object findObject(KeyInfo keyInfo, boolean isCreate, boolean generateCall
     final Object key = keyInfo.getKey();
```

Comment:
Original initialized null value is redundant; you may want to remove it as well.